setup/start.sh

```diff
@@ -147,7 +64,8 @@ source setup/zpush.sh
 source setup/management.sh
 
 # Ping the management daemon to write the DNS and nginx configuration files.
-while [ ! -f /var/lib/mailinabox/api.key ]; do
+until nc -z -w 4 localhost 10222
```

Comment:
In some case, there's a timing issue. `api.key` exists, but webservice is still unavailable because the service is currently starting.